api/v1/resourceslice.go

```diff
@@ -20,9 +20,9 @@ type ResourceSlice struct {
 }
 
 type ResourceSliceSpec struct {
-	CompositionGeneration int64      `json:"compositionGeneration,omitempty"`
+	CompositionGeneration int64      `json:"compositionGeneration,omitempty"` // deprecated
```

Comment:
😮 apiserver does not reject requests with unregistered fields right?

Reply:
For when we eventually remove these that is.

Reply:
Right- it returns warnings but not 400s